testing/resources/analysis-templates/scatterplot

```diff
@@ -0,0 +1,13 @@
+import matplotlib.pyplot as plt
```

Comment:
Each analysis should be self-contained. So you should have this import here also: `from js import data_object`

Also, start the section with a `%% py`.